app/styles/all.scss

```diff
@@ -21,8 +21,125 @@ $color_nhsuk-green--shadow: #00401e;
   float: right;
   font-weight: 600;
   padding: 6px 8px;
+  margin-right: -24px !important;
```

Comment:
can you change sizes to use nhsuk-spacing()

Reply:
these are very specific to this screen

Reply:
There are a fair bit of hardcoded px sizes here for margins and paddings. How does the screen look when it is resized?
Can we try and get as close as possible with the nhsuk-spacing?

Reply:
The nhsuk-spacing functions don't accommodate for every scenario but I can add them in in some places